docs/src/tutorials/tut_setup.md

```diff
@@ -1,13 +1,320 @@
+```jldoctest tut_setup; output = false
+sound_speed = 10.0
+state_equation = StateEquationCole(; sound_speed, reference_density=fluid_density,
+                                   exponent=7)
+
+# output
+StateEquationCole{Float64, false}(10.0, 7.0, 1000.0, 0.0)
+```
+The speed of sound here is numerical and not physical.
+We artificially lower the speed of sound, since the physical speed of sound
+in water would lead to prohibitively small time steps.
+The speed of sound in Weakly Compressible SPH should be chosen as small as
+possible for numerical efficiency, but large enough to limit density fluctuations
+to about 1%.
+
+TrixiParticles.jl requires the initial particle positions and quantities in
+form of an [`InitialCondition`](@ref).
+Instead of manually defining particle positions, you can work with our
+pre-defined setups.
+Among others, we provide setups for rectangular shapes, circles, and spheres.
+Initial conditions can also be combined with common set operations.
+See [this page](@ref initial_condition) for a list of pre-defined setups
+and details on set operations on initial conditions.
+
+Here, we use the [`RectangularTank`](@ref) setup, which generates a rectangular
+fluid inside a rectangular tank, and supports a hydrostatic pressure gradient
+by passing a gravitational acceleration and a state equation (see above).
+```jldoctest tut_setup; output = false, filter = r"RectangularTank.*"
+tank = RectangularTank(fluid_particle_spacing, initial_fluid_size, tank_size,
+                       fluid_density, n_layers=boundary_layers,
+                       acceleration=(0.0, -gravity), state_equation=state_equation)
+
+# output
+RectangularTank{2, 4, Float64}(InitialCondition{Float64}(...))
+```
+
+## Fluid system
+
+To model the water column, we use the Weakly Compressible Smoothed Particle
+Hydrodynamics (WCSPH) method.
+This method requires a smoothing kernel and a corresponding smoothing length,
+which should be chosen in relation to the particle spacing.
+```jldoctest tut_setup; output = false
+smoothing_length = 1.2 * fluid_particle_spacing
+smoothing_kernel = SchoenbergCubicSplineKernel{2}()
+
+# output
+SchoenbergCubicSplineKernel{2}()
+```
+You can find an overview over smoothing kernels and corresponding smoothing
+lengths [here](@ref smoothing_kernel).
+
+For stability, we need numerical dissipation in form of an artificial viscosity
+term.
+```jldoctest tut_setup; output = false
+viscosity = ArtificialViscosityMonaghan(alpha=0.02, beta=0.0)
+
+# output
+ArtificialViscosityMonaghan{Float64}(0.02, 0.0, 0.01)
+```
+We choose the parameters as small as possible to avoid visible viscosity,
+but as large as possible to stabilize the simulation.
+
+The WCSPH method can either compute the particle density by a kernel summation
+over all neighboring particles (see [`SummationDensity`](@ref)) or by making
+the particle density a variable in the ODE system and integrating it over time.
```

Comment:
Maybe mention that the former case calculates the density directly where as the latter rather computes the change of the density over time?